source/management_and_operations/references/template.rst

```diff
@@ -170,6 +170,9 @@ The OS system is defined with the ``OS`` vector attribute. The following sub-att
 | ``UUID``         | unique ID of the VM. It's referenced as machine ID inside the VM.  | O                | \-      | \-      | \-          |
 |                  | Could be used to force ID for licensing purposes.                  |                  |         |         |             |
 +------------------+--------------------------------------------------------------------+------------------+---------+---------+-------------+
+| ``FIRMWARE``     | firmware type (BIOS/EFI) or path                                   | O                | O       | \-      | \-          |
```

Comment:
we need to detail that for vCenter only bios/efi is valid, but not path

Reply:
changed